cmd/kubelet/app/server.go

```diff
@@ -213,9 +215,9 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 
 			// Config and flags parsed, now we can initialize logging.
 			logs.InitLogs()
-			logOption := &logs.Options{Config: kubeletConfig.Logging}
-			if err := logOption.ValidateAndApply(utilfeature.DefaultFeatureGate); err != nil {
-				return fmt.Errorf("failed to initialize logging: %v", err)
+			if err := kubeletConfig.Logging.ValidateAndApplyAsField(utilfeature.DefaultFeatureGate, field.NewPath("logging")); err != nil {
+				klog.ErrorS(err, "Failed to initialize logging")
+				os.Exit(1)
```

Comment:
why os.Exit instead of returning an error?

Reply:
I think that comes from the initial implementation where this code ran in a pre-function which didn't have an error return code. That was changed later. I'll update it.

Reply:
Done.